quick_sort.py

```diff
@@ -0,0 +1,40 @@
+import timeit
+import random
+
+
+def q_sort(self):
```

Comment:
Using `self` as the name of a parameter when you aren't using OO programming is deeply confusing.  Don't do it.